app/assets/javascripts/SourceCodeGlower/AnnotationTextDisplayer.js

```diff
@@ -72,6 +72,7 @@ AnnotationTextDisplayer.prototype.hide = function() {
 // Show the displayer
 AnnotationTextDisplayer.prototype.show = function() {
   this.display_node.style.display = 'block';
+  reloadDOM();
```

Comment:
I haven't played around with the Hound JS configuration, so... yeah, you can ignore the Hound comments for JS.